canal/sync.go

```diff
@@ -77,8 +77,8 @@ func (c *Canal) runSyncBinlog() error {
 		case *replication.RowsEvent:
 			// we only focus row based event
 			err = c.handleRowsEvent(ev)
-			if err != nil && (errors.Cause(err) != schema.ErrTableNotExist && errors.Cause(err) != schema.ErrMissingTableMeta) {
-				// if error is not ErrTableNotExist or ErrMissingTableMeta, stop canal
+			if err != nil && (errors.Cause(err) != ErrExcludedTable && errors.Cause(err) != schema.ErrTableNotExist && errors.Cause(err) != schema.ErrMissingTableMeta) {
```

Comment:
```
e := errors.Trace(err) 
// use e here
```

Reply:
Btw, the condition line is too long, you can use multi-lines.

Reply:
done